packages/components/src/components/notification/notification.tsx

```diff
@@ -164,24 +170,22 @@ export class Notification {
             `variant-${this.variant}`,
             this.isOpen && 'open'
           )}
-          role={this.role}
+          role={this.innerRole}
         >
           <div part="icon" aria-hidden="true">
             <slot name="icon">
               <IconTag size={ICON_SIZE} selected={this.type === 'toast'} />
             </slot>
           </div>
-          <div
-            part="body"
-            aria-live={this.role === undefined ? undefined : this.innerAriaLive}
-          >
+          <div part="body" role={this.innerRole} aria-live="polite">
             <div
               part="heading"
               role="heading"
               aria-level={this.headingLevel}
               aria-label={`${this.ariaHeading} ${this.heading}`}
             >
               <span>{this.heading}</span>
```

Comment:
Maybe this `span` should only be rendered if `this.heading` has a truthy value. When using the slot instead this would guarantee that no empty `span` is rendered inside the heading.

Reply:
i added this